docs/authentication/authentication.mdx

```diff
@@ -37,125 +36,149 @@ import AuthenticatorTypeTable from '../includes/_authenticator-type-table.mdx';
 
 import { Badge } from "@nextui-org/react";
 
-Currently the following authentication factors are supported and more are coming soon:  
+Currently the following authentication factors are supported and more are coming soon:
 
 - Universal Passkey
 
 - FIDO2 Multi-Device Passkey - <Badge color="primary" variant="flat" >Coming soon!</Badge>
 
 - Email OTP - <Badge color="primary" variant="flat" >Coming soon!</Badge>
 
-- Federated IDP - <Badge color="primary" variant="flat" >Coming soon!</Badge> 
- 
-
+- Federated IDP - <Badge color="primary" variant="flat" >Coming soon!</Badge>
 
-
-## Hosted Web 
+## Hosted Web
 
 The Hosted Web Authenticator is a constrained version of the Embedded SDK authenticator type. It requires less configuration and code, but it does require redirecting your users to a web-based authenticator hosted by Beyond Identity to utilize your application's passkeys on a user's device. The Hosted Web Authenticator is a Beyond Identity hosted page that embeds the Javascript SDK. Think of this as a wallet hosted by Beyond Identity Cloud that holds an identity's passkeys.
 
-
 Hosted Web Authenticator is a constrained version of the Embedded Authenticator Config with the following default values:
 
 <ul><li><b>Invocation Type</b>: automatic </li><li><b>Invoke URL</b>: https://auth-&#123;REGION&#125;.beyondidentity.com/authenticator/</li><li><b>Trusted Origin</b>: [https://auth-&#123;REGION&#125;.beyondidentity.com/authenticator/]</li></ul>
 
-
 :::tip KEEP IN MIND
 Passkeys are tied to an origin and they are stored on a specific browser.
 :::
```

Comment:
The passkey part is out of place too. We can probably remove this. It might be useful somewhere else
```suggestion
```